sunpy/net/dataretriever/client.py

```diff
@@ -179,6 +179,12 @@ def post_search_hook(self, exdict, matchdict):
                 rowdict[k] = exdict[k]
         return rowdict
 
+        for code in format_codes:
+            if code in replacements:
+                parse_pattern = parse_pattern.replace(code, replacements[code])
+
+        return parse_pattern
```

Comment:
How does this work?

Reply:
that was left in there by mistake, fixed it :100: